daemon/api_quota.go

```diff
@@ -0,0 +1,163 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
for consistency this should be called api_quotas.go (and same for the other related files)